content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -14,7 +14,7 @@ Follow this guide to upgrade and configure an ambient mode installation using
 [Helm](https://helm.sh/docs/). This guide assumes you have already performed an [ambient mode installation with Helm](/docs/ambient/install/helm-installation/) with a previous version of Istio.
 
 {{< warning >}}
-In contrast to sidecar mode, ambient mode supports moving application pods to an upgraded data plane without a mandatory restart or reschedule of running application pods. However, upgrading the data plane **will** briefly disrupt all workload traffic on the upgraded node, and ambient mode does not currently support canary upgrades of the data plane.
+In contrast to sidecar mode, ambient mode supports moving application pods to an upgraded data plane without a mandatory restart or reschedule of running application pods. However, upgrading the ztunnel **will** briefly disrupt all workload traffic on the upgraded node, and ambient mode does not currently support canary upgrades of the ztunnel.
```

Comment:
I almost regret asking, but should we follow https://github.com/istio/istio.io/issues/15232 and say `node proxy` instead of `ztunnel` here? Or maybe say `ztunnel` instead of `the ztunnel`?
 
 At least if we do namedrop it, let's use `<gloss>` so people can figure out what the heck we're talking about (I know we explicitly say this later on, but we should have a central definition for jargon and define them once, or use non-jargon)

Reply:
I've added a gloss reference to the ztunnel step.  I also have links to the ambient architecture page where these terms are explained at length.  While I support using the term L4 Node Proxy, I don't think this PR is the right place to do that work.